pluto_build/bin/config.sh

```diff
@@ -136,4 +137,24 @@ function import_public {
   psql -1 $BUILD_ENGINE -c "ALTER TABLE $name ADD COLUMN v text; UPDATE $name SET v = '$version';"
 }
 
+function import_qaqc {
+  name=$1
+  DO_folder=$2
+  target_dir=$(pwd)/.library/qaqc
+  qaqc_do_url=https://nyc3.digitaloceanspaces.com/edm-publishing/db-pluto/$DO_folder/output/qaqc
+  if [ -f $target_dir/$name.sql ]; then
+    echo "✅ $name.sql exists in cache"
+  else
+    echo "🛠 $name.sql doesn't exists in cache, downloading ..."
+    echo "{$url}/{$name.sql}"
+    mkdir -p $target_dir && (
+      cd $target_dir
+      rm $name.sql
+      curl -ss -O $qaqc_do_url/$name.sql
+    )
+  fi
+  psql $BUILD_ENGINE -c "DROP TABLE $name"
+  psql $BUILD_ENGINE -v ON_ERROR_STOP=1 -q -f $target_dir/$name.sql
+}
+
 register 'import' 'dataset' 'import given dataset to BUILD_ENGINE' import_public
```

Comment:
I am getting a `bin/config.sh: line 160: register: command not found` for anytime the register command is called in the config.sh - it doesn't seem to impact the dataloading but was curious if anyone else saw this as well? Is this a relic of the ancient version of postgres yall were discussing?

Reply:
I've seen that error message since I started working here and always ignored it. I think it's from broken bash code, not postgres

Reply:
I am getting it every time and it seems like nothing is impacted, so I just ignore.